src/comon/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,51 @@
+import {
+  StyledBox,
+  StyledNavLink,
+  StyledItem,
+  StyledText,
+} from './Sidebar.styled';
+import { ICONS } from '../../ui-kit/icons';
+import { useSidebarContext } from './SidebarContext';
+
+const Sidebar = () => {
+  const { showText } = useSidebarContext();
+
+  return (
+    <StyledBox showText={showText}>
+      <StyledNavLink to="/">
```

Comment:
Put all the routers in the constants folder so that you don't have to write them every time, but take them from the routers object